composer.json

```diff
@@ -28,7 +28,8 @@
         "psr/http-factory": "^1.0",
         "psr/http-server-handler": "^1.0",
         "psr/http-server-middleware": "^1.0",
-        "dflydev/fig-cookies": "^2.0"
+        "dflydev/fig-cookies": "^2.0",
+        "symfony/proxy-manager-bridge": "^4.0 || ^5.0"
```

Comment:
without this impossible reset doctrine manager

Reply:
I see! 

But we cannot force users to install this, if they don't use Doctrine, they may not need it. 

We need to find another solution.

Maybe in the middleware after the response yield, we could check if the manager has been initialized, and if it's closed, throw something like a `KernelNotReusableException` that the worker will catch and reboot the kernel for the next request?

What do you think?

Reply:
Hey, sorry for long reaction.

This check already done here - https://github.com/symfony/doctrine-bridge/blob/master/ManagerRegistry.php#L54, so I simple moved `symfony/proxy-manager-bridge` to suggestions, and add some info to README, in case if `symfony/proxy-manager-bridge` not used in project, it will throw exception, that stop worker.